lib/src/file_manager_logic.dart

```diff
@@ -48,18 +52,19 @@ class FileManagerLogic {
   ///
   /// The [updateUIFunc] returns a [File] object and the [uploadId] so that
   /// the UI can be refreshed with the updated values after uploading the file
-  static Future<void> selectFileFromGallery({
+  static Future<void> selectFileFromDevice({
     required BuildContext context,
     required UploadReturnId uploadAndReturnIdFunction,
     required Function toggleUpload,
     required String fileTitle,
     required void Function(File file, String uploadId) updateUIFunc,
+    bool? isFromCamera = false,
```

Comment:
Prefer passing the `ImageSource` directly instead of a `bool`

Reply:
Noted. Working on it.